classloader-leak-prevention/classloader-leak-prevention-core/src/main/java/se/jiderhamn/classloader/leak/prevention/preinit/Java2dRenderQueueInitiator.java

```diff
@@ -0,0 +1,26 @@
+package se.jiderhamn.classloader.leak.prevention.preinit;
+
+import se.jiderhamn.classloader.leak.prevention.ClassLoaderLeakPreventor;
+import se.jiderhamn.classloader.leak.prevention.PreClassLoaderInitiator;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+/**
+ * Using the class sun.java2d.opengl.OGLRenderQueue will spawn a new QueueFlusher thread with the same contextClassLoader.
+ */
+public class Java2dRenderQueueInitiator implements PreClassLoaderInitiator {
+    @Override
+    public void doOutsideClassLoader(ClassLoaderLeakPreventor preventor) {
+        Method getInstance = preventor.findMethod("sun.java2d.opengl.OGLRenderQueue", "getInstance");
+        if (getInstance != null) {
+            try {
+                getInstance.invoke(null);
+            } catch (IllegalAccessException e) {
```

Comment:
Since we won't want this library to create any issues that aren't already there, in most cases I catch `Throwable` or at least `Exception`. Any reason not to in this case?